tests/Makefile

```diff
@@ -3,7 +3,7 @@
 # DO NOT UPDATE THIS until you have published downstream builds for the kubebuilder tools, and checked
 # that there is an equivalent upstream version as well. Check upstream at https://storage.googleapis.com/kubebuilder-tools.
 # Publish downstream with `make -C tools publish-kubebuilder-tools`, see help text there for updating flags before publshing.
-ENVTEST_K8S_VERSION = 1.31.1
+ENVTEST_K8S_VERSION = 1.31.2
```

Comment:
Do we need to change it in this PR?

Reply:
Now that https://github.com/openshift/api/pull/2110 has been merged, it is not required anymore. I've removed it.